docker-compose.yml

```diff
@@ -79,3 +79,5 @@ services:
       - app:www.search.gov
       - app:ads.18f.gov
       - app:usability.gov
+      - app:developer.login.gov
+      - app:developers.login.gov
```

Comment:
I think you only add the original domain here

Reply:
Ok! Gave that a shot in 3cfb6c00cd0c57f08edc85b3973d33cebaf2200f